Platform/AlderlakeBoardPkg/CfgData/CfgData_BootOption_Azb.yaml

```diff
@@ -11,10 +11,9 @@
 - $ACTION      :
     page         : OS
 # usb
-- !expand { BOOT_OPTION_TMPL : [ 0 ,   0       ,  0 ,    5   ,   0   ,    0xFF  ,     1 ,     1 , '/boot/sbl_os' ] }
-- !expand { BOOT_OPTION_TMPL : [ 1 ,   0       ,  0 ,    5   ,   1   ,    0xFF  ,     1 ,     1 , '/boot/sbl_os' ] }
+- !expand { BOOT_OPTION_TMPL : [ 0 ,   0       ,  0 ,    5   ,   1   ,    0xFF  ,     1 ,     1 , '/boot/sbl_os' ] }
```

Comment:
While less common, I think that this will miss USB2 flash drives and the USB eMMC as a boot device.

Reply:
From the author comment, it is to WA a board issue. This is just a temp fix before a formal PR.